Library/Formula/gdk-pixbuf.rb

```diff
@@ -51,17 +68,21 @@ def install
 
   def post_install
     # Change the version directory below with any future update
-    ENV["GDK_PIXBUF_MODULEDIR"]="#{HOMEBREW_PREFIX}/lib/gdk-pixbuf-2.0/2.10.0/loaders"
+    if build.with?("relocations")
+      ENV["GDK_PIXBUF_MODULE_FILE"]="#{HOMEBREW_PREFIX}/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache"
```

Comment:
Note that with `GDK_PIXBUF_MODULE_FILE` set to the linked path, running `gdk-pixbuf-query-loaders` like this will delete the symlink and replace it with a regular file, [as reported here](https://github.com/Homebrew/homebrew/pull/44262#discussion_r40289868).  This causes an upgrade or an `unlink`/`link` to throw an error, as the regular file is not removed after an `unlink` or `rm` of a formula.  My solution is to set `GDK_PIXBUF_MODULE_FILE` to point at the `#{lib}/` path, rather than `#{HOMEBREW_PREFIX}/lib` path for this particular operation.